shop-be/import-service/src/services/stream.ts

```diff
@@ -0,0 +1,51 @@
+import { Product } from "src/types/product"
+import { Transform } from "stream"
+import { productServices } from "./db"
+import { sqsServices } from "./sqs"
+import { SQSClient } from "@aws-sdk/client-sqs";
+import { config } from '../../config'
+
+const { REGION } = config;
+
+export const stream = {
+    outputToQueue: () => {
+        const client = new SQSClient({ region: REGION });
```

Comment:
Such kind clients better initialise outside the function in sake of cachning
Please see - https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html (Take advantage of execution environment reuse to improve the performance of your function)